### PR TITLE
Define communication channels

### PR DESCRIPTION
**Pull request by pavelkumbrasev**
### Description 
State proper places for communication in main README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,10 +51,12 @@ You can get involved in this project in following ways:
 ## Support
 See our [documentation](./SUPPORT.md) to learn how to request help.
 
-## How to Contribute
+## Contribution guidelines
```
Comment (aepanchi) [inline README.md:54]: I don't think this change is necessary

File: README.md [hunk 2]
```diff
@@ -51,10 +51,12 @@ You can get involved in this project in following ways:
...
 We welcome community contributions, so check our [Contributing Guidelines](CONTRIBUTING.md)
 to learn more.
 
+Use GitHub Issues for feature requests, bug reports, and minor inquiries. For broader questions and development-related discussions, please turn to GitHub Discussions.
```
Comment (aepanchi) [inline README.md:58]:
```suggestion
Use GitHub Issues for feature requests, bug reports, and minor inquiries. For broader questions and development-related discussions, use GitHub Discussions.
```

File: README.md (hunk 1 above)
Comment (pavelkumbrasev) [inline README.md:54]:
```suggestion
## How to Contribute
```

**Review by aepanchi — APPROVED** [review]
LGTM

**Pull request merged**